website_event_compassion/wizards/event_registration_config_wizard.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, models, fields
+
+
+class EventRegistrationConfigWizard(models.TransientModel):
+    _name = 'event.registration.config.wizard'
```

Comment:
Rename the object like I proposed above `event.registration.communication.wizard`. Don't forget to update all the references in the code and in the views.